lib/safprofile.js

```diff
@@ -0,0 +1,99 @@
+const ZOWE_PROFILE_NAME_LEN = 246;
+const DEFAULT_INSTANCE_ID = "0";
```

Comment:
0 here rather than DEFAULT.
In the case that an instance ID isn't known, a default is needed... 0 or DEFAULT is fine.. but they should match, and match with documentation examples too.